### ISSUE TITLE
Request Snippet was not formed Properly - Swift

### ISSUE THREAD

**Issue by jeffayan** [a85/POSTMan-Chrome-Extension#4101]
//MARK:-  In Below code postData serialization snippet from the 
body was missing. it was not formed on request snippet.

import Foundation

let headers = [
  "content-type": "multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW",
  "x-requested-with": "XMLHttpRequest",
  "authorization": "Bearer jA1YmU5NGNlMmNmM2EyODYiLCJpYXQiOjE1MTEzNTYzNjUsIm5iZiI6MTUxMTM1NjM2NSwiZXhwIjoxNTEyNjUyMzY1LCJzdWIiOiIxMjQiLCJzY29wZXMiOltdfQ.ryeMBQjOWE5Qx9oBbQFHtutvBJXO93HdFQ3VQRS65lY28cl6vgRvyb3-g1niklMM4cRRWDjJU1_OaDNiM7pkl0rvikYtOJikG2nXkcdJKb9IV1R3n8dYdTwJcfErp21BwYVJzzUPE27HSLzIveirdYdFP32hAV4m8CpUJhH89WdcKvLZ1TyWFoQHxgH0l1lFQxGkPRts9ubwVbBF4Rgb9wZ8bS3pGkkMDncR6qkQ2DYBL_lFpEWUzMn35NblXi9B_1I8JmJumD7cBpux9SiPMqikJEOMSQoZ2JspZWG_mD8T4AjkPx3LNbPSq4RnNldLjnfwsgB-vqInHqM5RIFOko7GDXa4ZlouqC5B4HEYVvgZEphN-3RXCVgySOHvf4XrCV-_k8Vk5bwbZ1JCx2eDUscXccgMTdKMot-2wfjpOTTQC1_y8dKsMD5sFipmSHWkgJrRjzZ8O-eGWUH3Oa4vcqklpNrD2z1Az_0ckEZwkGqhK5-KYYeA8fanoOq5pRY-0RddUTp-uLPJDdcGw8PmF1DzYE2Mk7SMsE6aYhpvh4PH1AJfCKA5zTT9U5SN5yx_v8fBQbhwqHTNSQXzaOcmta7zho1FthPoEVYqrymCTAlHObtGZZgdOLUDl5jf0tPhFDfX9EjoeBV9Y5JvarQpNavOwR5rVoBalEFH6JLqcz0",
  "cache-control": "no-cache",
  "postman-token": "2f5502c0-2703-8cad-923d-e5ea0cef47a3"
]
let parameters = [
  [
    "name": "first_name",
    "value": "arun"
  ],
  [
    "name": "last_name",
    "value": "arun"
  ],
  [
    "name": "email",
    "value": "arun@XXXX.com"
  ],
  [
    "name": "mobile",
    "value": "1234567890"
  ],
  [
    "name": "picture",
    "fileName": ""
  ],
  [
    "name": "device_token",
    "value": "123456"
  ]
]

let boundary = "----WebKitFormBoundary7MA4YWxkTrZu0gW"

var body = ""
var error: NSError? = nil
for param in parameters {
  let paramName = param["name"]!
  body += "--\(boundary)\r\n"
  body += "Content-Disposition:form-data; name=\"\(paramName)\""
  if let filename = param["fileName"] {
    let contentType = param["content-type"]!
    let fileContent = String(contentsOfFile: filename, encoding: String.Encoding.utf8)
    if (error != nil) {
      print(error)
    }
    body += "; filename=\"\(filename)\"\r\n"
    body += "Content-Type: \(contentType)\r\n\r\n"
    body += fileContent
  } else if let paramValue = param["value"] {
    body += "\r\n\r\n\(paramValue)"
  }
}

let request = NSMutableURLRequest(url: NSURL(string: "http://XXXXXX/api/user/update/profile")! as URL,
                                        cachePolicy: .useProtocolCachePolicy,
                                    timeoutInterval: 10.0)
request.httpMethod = "POST"
request.allHTTPHeaderFields = headers
request.httpBody = postData as Data

let session = URLSession.shared
let dataTask = session.dataTask(with: request as URLRequest, completionHandler: { (data, response, error) -> Void in
  if (error != nil) {
    print(error)
  } else {
    let httpResponse = response as? HTTPURLResponse
    print(httpResponse)
  }
})

dataTask.resume()

<img width="1068" alt="screen shot 2018-01-24 at 11 46 58 pm" src="https://user-images.githubusercontent.com/17008251/35349899-3f150fe0-0162-11e8-8c00-7c0161bdd919.png">

**Comment by sdnts** [comment]
This seems to be a problem with the underlying library we use to generate this code. We'll look into this.

**Comment by akshitzaveri** [comment]
+1

**Comment by shreys7** [comment]
@jeffayan In Postman 7.2.1, we've introduced bug-fixes, support for two new languages (JS-Fetch and Powershell), and the ability to configure certain aspects of the generated snippet with the 'Experimental Codegen Mode' flag enabled in Postman's global settings. 

This particular bug has been solved in experimental codegens. Switch on the experimental codegen mode from settings and select Swift - URLSession.
![Screenshot 2019-07-08 at 7 57 18 PM](https://user-images.githubusercontent.com/20665864/60818142-b0942a80-a1ba-11e9-8cbf-17260b27b506.png)
